src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const inputsNodes = document.querySelectorAll('.field-text');
+
+for (const input of inputsNodes) {
+  const label = document.createElement('label');
+
+  label.className = 'field-label';
+
+  label.htmlFor = input.id;
+
+  label.textContent = input.name.toUpperCase();
```

Comment:
```suggestion
  label.textContent = input.name;
```
`.toUpperCase()` можно убрать, так как буквы будут большими по дефолту, в остальном прекрасно. Отдельный плюсик за before, как-то проще понимать где станет label. 👍🏻